### DISCUSSION TITLE
Possible problems from Twitch?

### DISCUSSION THREAD

**Discussion by albertopasqualetto** [bttv-android/bttv#187] [General]
Hello, I was wandering if there was any possibilty to be banned by Twitch because of using this mod?

Is the use of this app a violation of ToS or something?

Thanks

**Comment by FoseFx** [comment]
**Disclaimer: This is not legal advice. I am not a lawyer.**

Building it yourself may or may not be in violation of the Twitch TOS (Sec.9.vii and Sec.9.viii) depending on your jurisdiction of residence.
I could not find anything forcing you to only access their service using their clients (and even then you could argue that the actual connection to Twitch is done by the original parts of the app).

Anyway Twitch reserves the right to ban anyone for anything that they consider "fraudulent use" (Sec.14).

In any case it is doubtful that they would go after the users.

**Reply by albertopasqualetto** [reply]
Ok, I understand. Thanks!
